### PR TITLE
refactor: replace icon with a permanent github link 

### PR DESCRIPTION
**Pull request by SiddheshKukade**
**What kind of change does this PR introduce?**
Refracted image path to make it visible on docs.talawa.io

**Issue Number:**

Fixes https://github.com/PalisadoesFoundation/talawa/issues/1731
https://github.com/PalisadoesFoundation/talawa-docs/issues/404
**Did you add tests for your changes?**
N/A

**Snapshots/Videos:**
N/A

**If relevant, did you update the documentation?**
N/A

**Summary**

**Does this PR introduce a breaking change?**
N/A

**Other information**
N/A

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by SiddheshKukade** [comment]
Sir this PR will definitely solve logo problem:
check the results here: https://github.com/SiddheshKukade/talawa/tree/logo-render-fix#readme
image

File: README.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 [💬 Join the community on Slack](https://join.slack.com/t/thepalisadoes-dyb6419/shared_invite/zt-1s3fehykg-n2wfvgapdRxMakOQs~U7PA)
 
- 
-[![N|Solid](https://github.com/PalisadoesFoundation/talawa/blob/81c0d49458e176e8968fa42c610d220519860e06/assets/images/talawa-logo-lite-200x200.png)](https://github.com/PalisadoesFoundation/talawa) 
+![talawa-logo-lite-200x200](https://user-images.githubusercontent.com/65951872/231709493-cb09e37a-8125-4679-85eb-5d1de3b58afa.png)
```
Comment (palisadoes) [inline README.md:5]: We should be using:
- https://raw.githubusercontent.com/PalisadoesFoundation/talawa/develop/assets/images/talawa-logo-lite-200x200.png

This is the true source of the image if you click through the repo and look at the raw source via the location https://github.com/PalisadoesFoundation/talawa/blob/develop/assets/images/talawa-logo-lite-200x200.png

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please update the location

**Review by noman2002 — DISMISSED** [review]
Can be merged after the suggested changes by @palisadoes .

File: README.md (hunk 1 above)
Comment (SiddheshKukade) [reply README.md]: Okay @palisadoes, I'll update the path.

**Comment by SiddheshKukade** [comment]
@palisadoes , @noman2002 The path is updated please take a look

**Review by palisadoes — APPROVED** [review]

**Pull request merged**